scripts/deploy_pr_space.py

```
@@ -0,0 +1,99 @@
+def main() -> None:
+    api = HfApi(token=os.environ["HF_TOKEN"])
+    source = os.environ["SOURCE_SPACE"]
+    org = source.split("/")[0]
+    target = f"{org}/{os.environ['PR_SPACE_SLUG']}"
+    docker_repo = os.environ["DOCKER_REPO"]
+    image_tag = os.environ["IMAGE_TAG"]
+
+    try:
+        api.space_info(target)
+        print(f"Space '{target}' already exists")
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)